### DISCUSSION TITLE
Can i disable auto-import feature?

### DISCUSSION THREAD

**Discussion by rtabulov** [vuejs/language-tools#438] [Q&A] [answered]
So in my project i have vite setup with vite-plugin-components, which injects import statements for componts in selected folder automatically. 
Can i somehow disable volar's auto-import feature i the auto-imported component is in the selected folder. 
Or maybe just disable auto-import totally?

**Comment by johnsoncodehk** [comment, marked as answer]
Not possible at the moment, but we can add a option for this. (I can transfer this discussion to feature request issue if you want)

**Reply by ghost** [reply]
In case someone in the future comes across this discussion: this is already possible thanks to  https://github.com/johnsoncodehk/volar/issues/440 (the links to this issue GitHub creates aren't very visible) :P

**Comment by rtabulov** [comment]
yes, please, make a feature request to disable this feature *totally* 🙏
but i think disabling in *selected folders* is not that important
thanks

**Comment by MURHAF-ELMASRI** [comment]
Any Update??

**Comment by titusdecali** [comment]
Using Nuxt here and autoimports (which are unneeded in my project) are a pain to deal with when they suddenly appear
